### ISSUE TITLE
doubts when reproducing the O&C&I to M test

### ISSUE THREAD

**Issue by belengs** [qianyuzqy/IADG#6]
Hi, I am trying to reproduce your results, especially the O&C&I to M evaluation test with the pre-trained model you published, but the results I obtained are too different, and most probably I'm doing something wrong. I appreciate any help you can give me or any checks to do during the test to verify that the execution I am doing is correct. Also, I want to clarify some questions:

I saw that as input I have to provide two files (list_pos.txt, list_neg.txt) but I have some doubts about the format of these resources. I write in each line: "path_image label", is right? Also, I used the label 0 if it is an attack or 1 if it is real. Is this correct or is the expected labeling different? On the other hand, dataset M contains videos, how many frames are used to obtain the final result?

Thanks in advance, any help I will be grateful. 
Best regards.

**Comment by qianyuzqy** [comment]
Hi, @belengs 

Thanks for your interest in our work.
I' 'd like to address your concerns as follows:

1) Firstly, I am very sorry that you used the wrong label. In our code, label 0 is real and label 1 is fake.
2) Secondly, for the M dataset, we use around 150 frames for each video to calculate the final result.

Best Regards,

**Comment by belengs** [comment]
Thank you for responding so quickly and I appreciate your corrections. 

I applied these changes and the results are much better but I still cannot replicate the scores shown in the paper. In this context, I have doubts about the input and I'm wondering if the input face-crop receives any type of alignment or if is it necessarily centered before the inference. Would you consider this crop to be right?

![attack_client055_laptop_SD_printed_photo_scene01_222](https://github.com/qianyuzqy/IADG/assets/28621070/5e98c7e1-e05d-4b67-b08c-93d4b7b4b649)

Thank you again in advance
Regards

**Comment by qianyuzqy** [comment]
Hi, @belengs ,

Sorry for the late reply.

I'd like to address your concerns as follows:

1) We don't use any type of alignment.

2) Yes, it is better to be centered except for some border cases.

Thank you very much for your questions.

Best Regards,

**Comment by Elijah-Yi** [comment]
> Thank you for responding so quickly and I appreciate your corrections.
> 
> I applied these changes and the results are much better but I still cannot replicate the scores shown in the paper. In this context, I have doubts about the input and I'm wondering if the input face-crop receives any type of alignment or if is it necessarily centered before the inference. Would you consider this crop to be right?
> 
> ![attack_client055_laptop_SD_printed_photo_scene01_222](https://private-user-images.githubusercontent.com/28621070/278236594-5e98c7e1-e05d-4b67-b08c-93d4b7b4b649.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2OTU0NDAsIm5iZiI6MTcwOTY5NTE0MCwicGF0aCI6Ii8yODYyMTA3MC8yNzgyMzY1OTQtNWU5OGM3ZTEtZTA1ZC00YjY3LWIwOGMtOTNkNGI3YjRiNjQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA2VDAzMTkwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlMDE3ODQ5ZTkyNDc3MGZhYzJkZWJhMjlhODNiNTZlYjRjOGVjZDg1NjY0OTAwM2RjZGNlZTkwZmY4NmMwZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lHfEL5pHdNr91qMY9mxwZo5mEVeoVaJxK3qsEEr-__0)
> 
> Thank you again in advance Regards
Hi，Have you reproduced the results? The results I reproduced are quite different from those in the paper. Currently don't know where the problem is. Need your help, thank you
